hypothesis-python/tests/nocover/test_from_type_recipe.py

```diff
@@ -0,0 +1,76 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis/
+#
+# Most of this work is copyright (C) 2013-2019 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import absolute_import, division, print_function
+
+import datetime
+import decimal
+import fractions
+import numbers
+import uuid
+
+import hypothesis.strategies as st
+from hypothesis import given
+
+
+def everything_except(excluded_types):
+    """Recipe copied from the docstring of ``from_type``"""
+    return (
+        st.from_type(type)
+        .flatmap(st.from_type)
+        .filter(lambda x: not isinstance(x, excluded_types))
+    )
+
+
+many_types = [
```

Comment:
Apparently one or more of these things is not a type on Python 2 :sob:

So... `[t for t in ... if isinstance(t, type)]`?  And add `isinstance(None, excluded_types)  # validate input` to the recipe.

Reply:
noooooooooo (will do)

:sob:

Reply:
...got to wait for the deploy, sorry :cry:

Reply:
@Zac-HD shouldn't I just remove those problematic types from the test? It isn't an issue with the recipe, just an issue with me feeding the recipe a type that isn't a type in Python 2.

Reply:
Yeah, the validation isn't really needed in the recipe.